README.md

```diff
@@ -288,4 +288,4 @@ Questions can be asked by raising an `Issue`.
 - [Vivek Singh](https://github.com/vivmost)
 - [Vladyslav Kyiashko](https://github.com/vlakyi)
 - [Yashasvi Misra](https://github.com/yashasvimisra2798)
-- [Yaswanth Modepalli](https://github.com/Yaswanth14)
```

Comment:
What happened here?

Reply:
No, not something serious. The `-` was not placed rightly in file but was displaying well but I just decided to do it properly!

Reply:
oh okay